modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -970,6 +970,11 @@ public long getCompactionRevision() {
         }
     }
 
+    @Override
+    public @Nullable Long checksum(long revision) {
+        throw new UnsupportedOperationException();
```

Comment:
Let's implement the method to be as universal as possible.

Reply:
Checksums are not needed for normal functioning of the Metastorage, they will only be used before starting it, when performing a special procedure to re-enter a node that did not see the Metastorage repair. This will be tested in integration tests, so only using the RocksDB-based storage. Hence we simply don't need checksums in the in-mem storage. That's why it makes no sense to implement checksums there.

If something changes in the future, we'll add this to this storage as well (but I doubt it will change).